integrations/react-ts/src/components/dropIn.tsx

```diff
@@ -95,21 +110,15 @@ const Index: React.FC = () => {
 
   return (
     <div>
-      <h2>DropIn integration</h2>
-      {/* Example below: show loading state */}
-      {!elementShow && <p>Loading...</p>}
-      {/* Example below: display response message block */}
-      {errorMessage.length > 0 && <p id="error">{errorMessage}</p>}
       {/**
-       * STEP #3a: Add an empty container for the dropin element to be placed into
+       * STEP #4a: Add an empty container for the dropin element to be placed into
        * - Ensure this is the only element in your document with this id,
        *   otherwise the element may fail to mount.
        */}
       <div
         id="dropIn"
         style={{
           ...containerStyle, // Example: container styling can be moved to css
-          display: elementShow ? 'block' : 'none', // Example: only show element when mounted
```

Comment:
setElementShow is useful for merchants, can we reserve this part?

Reply:
Drop-in has built-in loading. So we can remove the elementShow state.